src/SourceGenerators/Uno.UI.Tasks/LinkerHintsGenerator/LinkerHintGeneratorTask.cs

```diff
@@ -259,6 +285,26 @@ private static List<string> FindAvailableLinkerHints(List<AssemblyDefinition> as
 			return hints.Distinct().ToList();
 		}
 
+		private static Dictionary<string, List<TypeDefinition>> BuildAvailableTypes(List<AssemblyDefinition> assemblySearchList)
+		{
+			Dictionary<string, List<TypeDefinition>> map = new();
+
+			foreach (var asm in assemblySearchList)
+			{
+				foreach(var type in asm.MainModule.Types)
+				{
+					if(!map.TryGetValue(type.FullName, out var list))
+					{
+						list = new();
```

Comment:
Should we add `type.FullName` and `list` to `map` at this point?

Reply:
We definitely should, good catch :) I have not tested this part yet, I wanted a CI package to test with.